.tmp/tasks/1.1-project-initialization.md

```
@@ -0,0 +1,530 @@
+
+`/api` 以下でREST APIを提供します。同じHonoアプリケーション内で実装します。
+
+```typescript
+app.get('/api/health', (c) => {
+  return c.json({ status: 'ok' })
+})
+```
+
+#### ルーティング構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)